Download.md

```diff
@@ -0,0 +1,15 @@
+
+# LearnGit
+
+Useful links to learn download git for windows
+
+* https://git-scm.com/download/win
+
+Useful links to learn download git for windows
+
+* https://git-scm.com/download/mac
+
+Useful links to learn download git for windows
```

Comment:
Kindly change to respective OS for the link.

Reply:
Resolved